spanner/system-test/spanner.test.js

```diff
@@ -53,103 +53,139 @@ test.after.always(async (t) => {
 
 // create_database
 test.serial(`should create an example database`, async (t) => {
-  const output = await tools.runAsync(`${schemaCmd} createDatabase "${INSTANCE_ID}" "${DATABASE_ID}"`, cwd);
-  t.true(output.includes(`Waiting for operation on ${DATABASE_ID} to complete...`));
-  t.true(output.includes(`Created database ${DATABASE_ID} on instance ${INSTANCE_ID}.`));
+  const results = await tools.runAsyncWithIO(`${schemaCmd} createDatabase "${INSTANCE_ID}" "${DATABASE_ID}"`, cwd);
+  const output = results.stdout + results.stderr;
+  t.regex(output, new RegExp(`Waiting for operation on ${DATABASE_ID} to complete...`));
+  t.regex(output, new RegExp(`Created database ${DATABASE_ID} on instance ${INSTANCE_ID}.`));
 });
 
 // insert_data
 test.serial(`should insert rows into an example table`, async (t) => {
-  let output = await tools.runAsync(`${crudCmd} insert ${INSTANCE_ID} ${DATABASE_ID}`, cwd);
-  t.true(output.includes(`Inserted data.`));
+  const results = await tools.runAsyncWithIO(`${crudCmd} insert ${INSTANCE_ID} ${DATABASE_ID}`, cwd);
+  const output = results.stdout + results.stderr;
+  t.regex(output, new RegExp(`Inserted data.`));
```

Comment:
Replace `t.regex(output, new RegExp($STRING))` with `t.regex(output, /$STRING/)`. Apply this comment throughout your PR.